### PR TITLE
Add palantir-sdk recipe

### PR DESCRIPTION
**Pull request by ahggns**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AlexandreGuinaudeau** [comment]
Confirming I'm willing to be listed as maintainer

**Comment by tom-s-powell** [comment]
Confirming I'm willing to be listed as maintainer

**Comment by ahggns** [comment]
@conda-forge/help-python, ready for review!

File: recipes/palantir-sdk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6652b20d0e35f53942c63a2dc5699603cc09f84eef20691d628cc22bdb0efcb1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (chrisburr) [inline recipes/palantir-sdk/meta.yaml:15]: These flags are handled automatically by environment variables these days:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```

File: recipes/palantir-sdk/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (chrisburr) [inline recipes/palantir-sdk/LICENSE:1]: This file isn't needed as it's in the souce files (you still need the key in `meta.yaml`).

**Comment by ahggns** [comment]
Thanks @chrisburr, made the requested changes should be ready for another review.

**Pull request merged**